docs/galaxy/overview.md

```diff
@@ -4,4 +4,6 @@ sidebar_position: 1
 ---
 # Galaxy
 
-to be updated
+Galaxy is the fundamental request entrypoint for all TAMU Datathon applications. The reason why we use Galaxy is to ensure that all requests are directed to the right applications. Galaxy is hosted on [Vercel](https://vercel.com/)
+
+Having a single application to take care of requests and redirecting allows us to efficiently deal with issues and create other projects that is not in the galaxy repository.
```

Comment:
There are no projects withing the galaxy repository. You can just say that galaxy allows us to efficiently deal with connecting different repositories under one domain.